package.json

```diff
@@ -6,33 +6,33 @@
   "bin": "lib/index.js",
   "bugs": "https://github.com/felipecrs/bindl/issues",
   "dependencies": {
-    "@felipecrs/decompress-tarxz": "^5.0.2",
-    "@xhmikosr/decompress-tar": "^8.0.1",
-    "@xhmikosr/decompress-tarbz2": "^8.0.2",
-    "@xhmikosr/decompress-targz": "^8.0.1",
-    "@xhmikosr/decompress-unzip": "^7.0.0",
-    "@xhmikosr/downloader": "^15.0.2",
-    "chalk": "^5.4.1",
+    "@felipecrs/decompress-tarxz": "^5.0.4",
+    "@xhmikosr/decompress-tar": "^8.1.0",
+    "@xhmikosr/decompress-tarbz2": "^8.1.0",
+    "@xhmikosr/decompress-targz": "^8.1.0",
+    "@xhmikosr/decompress-unzip": "^7.1.0",
+    "@xhmikosr/downloader": "^15.2.0",
+    "chalk": "^5.5.0",
     "clipanion": "^4.0.0-rc.4",
     "cosmiconfig": "^9.0.0",
     "execa": "^9.6.0",
-    "listr2": "^9.0.0",
+    "listr2": "^9.0.1",
     "rimraf": "^6.0.1"
   },
   "devDependencies": {
-    "@types/node": "^20.17.10",
+    "@types/node": "^22.17.0",
```

Comment:
I keep types/node at the minimum version of node my project supports, so that if I try to use some function that is available only in newer node versions, it will error right in my editor.

(`import.meta.parent` is one example)